src/styles/blocks/_table.scss

```diff
@@ -0,0 +1,45 @@
+table {
+  text-align: left;
+  width: 100%;
+
+  tr {
+    border-bottom: 1px solid var(--gcds-border-default);
+  }
+  tr:first-child {
+    border-bottom: 2px solid var(--gcds-border-default);
+  }
+
+  td,
+  th {
+    padding: var(--gcds-spacing-300) var(--gcds-spacing-250);
+    &:first-child {
+      padding-left: 0;
+    }
+    &:last-child {
+      padding-right: 0;
+    }
+  }
+
+  th {
+    font-weight: var(--gcds-font-weights-medium);
+    padding-top: 0;
+  }
+
+  // Column width templates
+  &.table-display-token-text,
+  &.table-display-token-link,
+  &.table-display-token-border,
+  &.table-display-token-danger,
+  &.table-display-token-disabled,
+  &.table-display-token-focus {
```

Comment:
Instead of listing each table, can we give this style a general class name that we can then add to each table that is supposed to use this style?

Reply:
That might be tricky. The tables are generated automatically, its easier to output a class using a key that exists in the data.

Would using the "type" instead of the token name be okay? we'd have less types of tables.